grails-app/migrations/views/stock-movement.sql

```diff
@@ -0,0 +1,131 @@
+CREATE OR REPLACE VIEW stock_movement AS
+    select
+           r.id,
+           r.name,
+           r.request_number as identifier,
+           r.description,
+           r.origin_id,
+           r.destination_id,
+           r.date_requested,
+           r.date_created,
+           r.last_updated,
+           r.requested_by_id,
+           r.created_by_id,
+           r.updated_by_id,
+           ri.item_count,
+           'OUTBOUND' as stock_movement_type,
```

Comment:
We probably want to change OUTBOUND and INBOUND to something that describes the stock movement type rather than the direction. I'll get back to you.